splitgpg2/__init__.py

```diff
@@ -315,13 +319,51 @@ def load_config(self, config) -> None:
             'verbose_notifications',
             'allow_keygen',
             'gnupghome',
+            'auto_sync',
             'isolated_gnupghome_dirs',
             # handled in main()
             'debug_log',
         )
         for option in config:
             if option not in supported_options:
                 self.log.warning('Unsupported config option: %s', option)
+        self.log.info('Using GnuPG home directory %s', self.gnupghome)
+        os.makedirs(self.gnupghome, 0o700, exist_ok=True)
+        if self.auto_sync:
+            import pwd
+            new_home_directory = os.path.join(self.gnupghome, 'qubes-auto-keyring')
+            try:
+                os.mkdir(new_home_directory, 0o700)
+            except FileExistsError:
+                pass
+            self.gnupghome = new_home_directory
+            default_gnupg_home = os.path.join(pwd.getpwuid(os.getuid()).pw_dir, '.gnupg')
```

Comment:
Shouldn't this use original `self.gnupghome` value instead? Together with reverting 06326e428f669d546f54fbdbd51e86227e36fe95 (as it was a workaround until this feature gets implemented).